browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -4052,6 +4052,9 @@ If you don't accept this request, VPN will not reconnect and your internet conne
     <message name="IDS_PRIVATE_TAB_SCREENSHOT_SUMMARY" desc="A summary to show private tab screenshot">
       If allowed, private tabs will also be visible when multiple apps are open.
     </message>
+    <message name="IDS_BRAVE_HOMEPAGE_MOBILE_BOOKMARKS_TEXT" desc="Text for Mobile Bookmarks setting in Home Page settings">
+      Mobile Bookmarks
```

Comment:
@samartnik can you add a screenshot with the new string, please. Thanks!

Reply:
Sure, here it is

![Screenshot_20250514_121346](https://github.com/user-attachments/assets/670a5d98-3bf3-41aa-b416-07d88011e22c)

Reply:
Looks like it should be `Mobile bookmarks`, I'll fix this one

Reply:
That's how it looks now

![Screenshot_20250514_122850](https://github.com/user-attachments/assets/9a73b181-8518-46fd-b2b4-4d63463c25a5)